src/Routing/CrudLoader.php

```diff
@@ -12,6 +12,9 @@
 
 class CrudLoader extends Loader
 {
+    // TODO: add patterns to support other versions e.g. ulid as well.
+    private const URL_GENERATOR_ENTITY_KEY_REQUIREMENT_EXPRESSION = '[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}|\d+';
```

Comment:
to support wide range of id we should allow to configure this regex in the bundles config.